pandas/core/groupby/groupby.py

```diff
@@ -1266,8 +1266,40 @@ def count(self):
     def mean(self, *args, **kwargs):
         """
         Compute mean of groups, excluding missing values
+        
```

Comment:
May be you can try adding long summary and Parameters as well, see https://python-sprints.github.io/pandas/guide/pandas_docstring.html